### PR TITLE
fix networking reference

### PR DESCRIPTION
**Pull request by arimal199**

### CODE REVIEW

File: engine/swarm/networking.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ A Docker swarm generates two different kinds of traffic:
 
 This topic discusses how to manage the application data for your swarm services.
 For more details about swarm networking in general, see the
-[Docker networking reference architecture](https://success.docker.com/Architecture/Docker_Reference_Architecture%3A_Designing_Scalable%2C_Portable_Docker_Container_Networks){: target="_blank" rel="noopener" class="_" }.
+[Docker networking reference architecture](https://docs.docker.com/network/){: target="_blank" rel="noopener" class="_" }.
```
Comment (thaJeztah) [inline engine/swarm/networking.md:18]: As we're now no longer leaving the docs website, I think it's ok to remove the `target=_blank`;

```suggestion
[Docker networking reference architecture](https://docs.docker.com/network/).
```

Comment (dockertopia) [reply engine/swarm/networking.md]: I'm fine with this change, but opening in another tab seems to be a behavior being used by us on other occasions other than when target link is for page outside Docker docs.

**Review by dockertopia — APPROVED** [review]
@arimal199 Thanks for the fix! 
@thaJeztah As agreed TOC fix following in a separate PR, thanks.

**Review by dockertopia — COMMENTED** [review]
This image is missing. I'm removing this sentence as remedy/compromise in a separate PR (#15069).

**Comment by dockertopia** [comment]
> @arimal199 Thanks for the fix! @thaJeztah As agreed TOC fix following in a separate PR, thanks.

PS - Your suggestion was added here (with some adjustments) #15069.

**Pull request closed without merging**